tusker/__init__.py

```diff
@@ -191,23 +201,29 @@ def cmd_diff(args, cfg: Config):
     target = args.target
     if args.reverse:
         source, target = target, source
-    sql = tusker.diff(source, target, with_privileges=args.with_privileges)
-    print(sql, end='')
+    try:
+        sql = tusker.diff(source, target, with_privileges=args.with_privileges)
+        print(sql, end='')
+    except Exception as e:
+        print(str(e))
 
 
 def cmd_check(args, cfg: Config):
     backends = args.backends
     if 'all' in backends:
         backends = ['migrations', 'schema', 'database']
     tusker = Tusker(cfg, args.verbose)
-    diff = tusker.check(backends, with_privileges=args.with_privileges)
-    if diff:
-        print('Schemas differ: {} != {}'.format(diff[0], diff[1]))
-        print('Run `tusker diff` to see the differences')
-        sys.exit(1)
-    else:
-        print('Schemas are identical')
-        sys.exit(0)
+    try:
+        diff = tusker.check(backends, with_privileges=args.with_privileges)
+        if diff:
+            print('Schemas differ: {} != {}'.format(diff[0], diff[1]))
+            print('Run `tusker diff` to see the differences')
+            sys.exit(1)
+        else:
+            print('Schemas are identical')
+            sys.exit(0)
+    except Exception as e:
```

Comment:
I'd prefer if you created a custom `TuskerError` instead so you don't over-catch. e.g. in case of a programming error I'd still like to get a full blown trace back and not just a plain and meaningless error message like `Key Error: 'foo'`

Reply:
Maybe `TuskerError` is a tad too generic. How about calling it  `ExecError` instead?

I just don't like using a plain `Exception` object here as it will now catch all sorts of `Exception`s and not just failed executions of SQL files.